lib/LaTeXML/Post/Manifest/Epub.pm

```diff
@@ -244,7 +264,22 @@ sub finalize {
     my $file_url  = URI::file->new($file);
     $file_item->setAttribute('id',         url_id($file_url));
     $file_item->setAttribute('href',       $file_url);
-    $file_item->setAttribute('media-type', $file_type); }
+    $file_item->setAttribute('media-type', $file_type);
+
+    if ($file_type eq 'image/svg+xml') {
+      my @properties;
+      my $doc;
+      eval {
+        $doc = XML::LibXML->load_xml(location => pathname_concat($OPS_directory, $file)); };
+      if ($@) {
+        Error('parsing', $file, undef, "$@"); }
+      else {
+        push @properties, 'scripted' if $XPATH->findnodes($xpath_scripted, $doc);
+        push @properties, 'mathml  ' if $XPATH->findnodes($xpath_mathml,   $doc);
+        my $properties = join(" ", @properties);
+        $file_item->setAttribute('properties', $properties) if $properties; } }
+
+  }
```

Comment:
This test is basically what we'd want to determine whether to use img or object, as discussed in #1444, right?  That suggests that it might be better higher up the chain, such as `Post::Graphics`?

Reply:
The img/object distinction is a bit more subtle because the `<img>` tag disables every external resource as well. I'd say this code is specific to EPUB (the spec just wants to know if there are scripts and MathML content*).

* Actually, one should also declare external resources, e.g. web fonts not embedded in the file. This requires parsing the CSS content of the image, so it seems quite hard to do reliably.

Reply:
What would be the harm in simply assuming that any svg image contains mathml and scripts?

Reply:
Per spec, the `mathml` and `scripted` properties "MUST be set if and only if the criterion specified in the description is met", so the output would be invalid. To my memory, epubcheck was flagging the properties as errors if no mathml/scripts were in the file.